operational/fnms/schedule_fnms_reports/README.md

```diff
@@ -1,46 +1,69 @@
-## Schedule FlexNet Manager report
+# Schedule FlexNet Manager report
+
+## What it does
 
-### What it does
 This policy will run a FlexNet Manager report (Custom view) and send the result via email.
 The policy is a report only policy, no action is taken during the Policy Escalation.
 
 The report / Mail output looks like this:
 ![Alt text][emailoutput]
 
 Current limitations:
-- This only works with the FNMS Cloud
+
 - Output is limited to max 100.000 rows.
 
-### Pre-reqs
-- FlexNet Manager
+## Prerequisites
+
+- FlexNet Manager.
+
+    __Note:__ If the FlexNet Manager Suite is not accessible from the Internet, you will need to setup a wtunnel to provide the necessary connection from the Policy engine to the FlexNet manager. For more details on wtunnel please refer to this: <https://github.com/rightscale/wstunnel>
```

Comment:
A few suggestions on this.  replace wtunnel with wstunnel.  Remove from the Policy engine.  Fix this link to wstunnel.  also run the paragram through word to check grammer.